src/storeConfig.js

```diff
@@ -40,14 +47,13 @@ export default function storeConfig(initialState, history) {
   }, appReducers));
 
   if (__DEV__ && !__SERVER__) {
-    if (__LOGREDUX__) middleware.push(createLogger());
     store = createStore(canonReducer, initialState, compose(
-      applyMiddleware(...middleware),
+      appliedMiddleware,
       typeof window === "object" && typeof window.devToolsExtension !== "undefined" ? window.devToolsExtension() : f => f
```

Comment:
the exposed function by redux devtools changed; this property will need to be updated in the future
https://github.com/zalmoxisus/redux-devtools-extension#usage

Reply:
@frabarz just pushed a tweak, what do you think?